README.md

```diff
@@ -1,4 +1,52 @@
-## Setup the Database
+# Running the App
+The repository contains Docker configuration which can be used out of the box, however, the app can be run without using
+the docker container. Both way are discussed below:
+
+## 1. Run inside Docker Container
+It is the recommended way, the Docker Image of the app is pre-configured to run the `migrations` and `collectstatic`.
+However, the `load` command to populate the database and the crons need to be run manually.
+
+The image can be built locally or can be pulled from Docker Hub:
+
+```
+docker pull arjunsalyan/macports-webapp
+```
+or build the image
+```
+docker build -t macports-webapp .
+```
+
+After the image has been build or pulled, run it using an an env file that contains environment variables. Create a file
+`env` in the root of the project and supply the following information.
+
+**env**: *(Contains environment variables)*
+```
+DB_NAME=
+DB_USER=
+DB_PASSWORD=
+DB_HOST=
+SECRET_KEY=
+```
+
+Now run the image:
+```
+docker run -d -p 80:80 --env-file=env macports-webapp
+```
+
+To start the cron jobs:
+
+```
+docker exec <container-id> supervisorctl start cron
+```
+All other commands supported by the app can be run using:
+```
+docker exec <container-id> [command]
+```
+where example of [command] may be: `python3 /code/app/manage.py load`.
+
+## 2. Run Without Docker
+The `/app` directory is a standalone Django-app which can be run normally like any other django application.
+### Setup the Database
 Enter your database credentials in MacPorts/settings.py
```

Comment:
This might have a consequence of the user ending up with merge conflicts, or accidentally committing some passwords.